target/linux/lantiq/patches-4.19/0712-NET-MIPS-lantiq-add-pcdu-settings.patch

```diff
@@ -0,0 +1,147 @@
+From 03cd3a82b39a43660f4b4f37738b7f4006c657d8 Mon Sep 17 00:00:00 2001
+From: Jonas Albrecht <plonkbong100@protonmail.com>
+Date: Fri, 24 Jan 2020 17:28:31 +0000
+Subject: [PATCH] net: lantiq_xrx200: Add pcdu settings
+X-Patchwork-Bot: notify
+
+This patch modifie:
+drivers/net/ethernet/lantiq_xrx200.c
+For
+Add support for for PCDU
+Fix typo in tx/rx delay check.
+Fix wrong port number report for fixed link.
+Fix printf format string.
+
+This is required for some Routers devices like Arcadyan VGV952CJW33-E-IR
+
```

Comment:
It seems to be related to RT3883 where @dangowrt seems to say it is already supported.

What happens if we do not apply this patch, or what does not work? I see other drivers taking delays from OF so it can probably be upstreamed. But it requires testing and understanding. So i ask if anyone knows what exactly it is needed for and how to test things.

Reply:
RT3883 is supported in OpenWrt -- as a SoC running Linux, if it has enough RAM connected to it. Here RT3883 seems to be used in "iNIC" mode, ie. it is connected to the Lantiq SoC which runs OpenWrt via RGMII and runs a minimal firmware which only needs the built-in SRAM of the chip so it does all the WiFi stuff. Using iNIC mode is *not* supported in OpenWrt at this point.

Reply:
Ok that sounds really bad, could be a show-stopper. I would consider WiFi support really needed, but the drivers i see do not look like anyone would want to use let alone maintain them.

I will still dump what i found from the new owner of the vendor

via https://www.mediatek.com/products/networking-and-connectivity/legacy-products

i found https://d86o2zu8ugzlg.cloudfront.net/mediatek-craft/drivers/DPO_RT5572_LinuxSTA_2.6.1.3_20121022.tar.bz2

which very much looks like https://github.com/Quallenauge/ralink_inic/ but in another version